compliance/fnms/fnms_licenses_expiring/README.md

```diff
@@ -1,6 +1,15 @@
 # ITAM Expiring Licenses
+Generates report of licenses expiring and sends the result as an email.
 
-## What it does
+## Prerequisites
+This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Automation/ManagingCredentialsExternal.htm) for authenticating to datasources -- in order to apply this policy you must have a Credential registered in the system that is compatible with this policy. If there are no Credentials listed when you apply the policy, please contact your Flexera Org Admin and ask them to register a Credential that is compatible with this policy. The information below should be consulted when creating the credential(s).
+
+- [**Flexera Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm) (*provider=flexera*) which has the following roles:
+  - `fnms_user`
```

Comment:
@cdwilhelm - Can you help me confirm what permissions are needed for these ITAM policies that use the `api.flexera.com/fnms/v1/orgs/....` endpoint please?  I assumed fnms_user or fnms_admin for these because they were not defined in the previous README

Reply:
I'm not going to be much help here.  The FNMS policies were written by solution enginers.  I would suggest connecting with an SE who is familiar with FNMS/Itam.